go/vt/dbconfigs/dbconfigs.go

```diff
@@ -135,69 +135,102 @@ func registerPerUserFlags(dbc *userConfig, userKey string) {
 
 }
 
+// ConnParams contains mysql.connparams for now
+type ConnParams struct {
+	connParams *mysql.ConnParams
+}
+
+// MakeConnParams is for tests only
+func MakeConnParams() ConnParams {
```

Comment:
Combine this with `SetConnParams` and rename to:
```
func New(mcp *mysql.ConnParams) ConnParams {
}
```

Reasons:
* I don't see a use case where you're using one without the other.
* Constructors should be called "New".